src/app/components/Search/Search.jsx

```diff
@@ -102,63 +112,50 @@ class Search extends React.Component {
   }
 
   render() {
     return (
-      <form
+      <SearchBar
         id="mainContent"
         onSubmit={this.triggerSubmit}
-        onKeyPress={this.triggerSubmit}
-        action={`${appConfig.baseUrl}/search`}
-        method="POST"
-        className="nypl-omnisearch-form"
-        aria-controls="results-description"
+        className="content-primary"
+        attributes={{
+          method: 'POST',
+          action: `${appConfig.baseUrl}/search`,
+        }}
       >
-        <div className="nypl-omnisearch">
-          <div className="nypl-text-field">
-            <span className="nypl-omni-fields">
-              <label htmlFor="search-by-field">Search in</label>
-              <select
-                ref={this.setSearchByFieldRef}
-                id="search-by-field"
-                onChange={this.onFieldChange}
-                value={this.state.field}
-                name="search_scope"
-              >
-                <option value="all">All fields</option>
-                <option value="title">Title</option>
-                <option value="journal_title">Journal Title</option>
-                <option value="contributor">Author/Contributor</option>
-                <option value="standard_number">Standard Numbers</option>
-                <option value="subject">Subject</option>
-              </select>
-            </span>
-          </div>
-          <div className="nypl-text-field">
-            <span className="nypl-omni-fields-text">
-              <label htmlFor="search-query" id="search-input-label" className="visuallyhidden">
-                Search Shared Collection Catalog for
-              </label>
-              <input
-                type="text"
-                id="search-query"
-                aria-labelledby="search-input-label"
-                aria-controls="results-description"
-                placeholder="Keyword, title, journal title, or author/contributor"
-                onChange={this.inputChange}
-                value={this.state.searchKeywords}
-                name="q"
-              />
-            </span>
-          </div>
-          <SearchButton
-            className="nypl-omnisearch-button nypl-primary-button"
-            onClick={this.submitSearchRequest}
-          />
-        </div>
-      </form>
+        <Select
+          id="search-by-field"
+          onChange={this.onFieldChange}
+          selectedOption={this.state.field}
+          name="search_scope"
+        >
+          <option value="all">All fields</option>
+          <option value="title">Title</option>
+          <option value="journal_title">Journal Title</option>
+          <option value="contributor">Author/Contributor</option>
+          <option value="standard_number">Standard Numbers</option>
+          <option value="subject">Subject</option>
+        </Select>
+        <Input
+          type="text"
+          id="search-query"
+          aria-labelledby="search-input-label"
```

Comment:
This is related to the test you deleted where it's checking for a label (`should render a label for the select element`), this should be `aria-label="Search by keyword, title, journal title, or author/contributor"`. The `aria-labelledby` would be fine if you had another element on the page with id `search-input-label` but there is none (as far as I can tell). The test you deleted is fine to delete but this should be updated.